docs/events-API-v1/03-Custom-Event-Transformer.md

```diff
@@ -86,11 +85,6 @@ The PagerDuty event payload is fairly simple. It is a JSON object in the same fo
 
 For convenience, constants are provided for the `event_type` field: `PD.Trigger`, `PD.Acknowledge`, and `PD.Resolve`.
 
-<Alert variant="info">
-  <Alert.Heading>Debug Mode</Alert.Heading>
-  Setting your Custom Event Transform to debug mode allows you to test your universal transform as you write it. If there are any errors in your code, you will still receive a triggered incident with a brief description of the error.<br/><br/>You can set the integration to debug mode by changing the configuration of the integration the service.
-</Alert>
-
```

Comment:
Fwiw, I'm not sure if CET debug mode relates to App Event Transform debug mode 🤔